### DISCUSSION TITLE
What is the status of XP 12 support?

### DISCUSSION THREAD

**Discussion by Simssmith** [kubilus1/autoortho#12] [General]
Will the latest version work in XP 12?

**Comment by kubilus1** [comment]
Technically, yes, this will work in XP 12, just like XP 11.  However, it appears there are some pretty serious performance regressions in the latest beta of XP12 (beta6) in general,  that affect how terrain is loaded.  AFAICT, this applies to all terrain packs/orthos, but will be noticeable using this tool.

The long and short is that in XP11, as XPlane reads a terrain file, it immediately begins to process and load DDS (ortho imagery).  This is efficient and allows for autoortho to process and load imagery as new terrain is being pulled in.
 
Xplane 12, however, will read the entire terrain information for a DSF file first, before beginning to read in or process and DDS imagery.  This slows down the entire terrain loading process significantly.  Not much I can do about it, but hopefully this is something that is addressed by Laminar Research when they do perf tweaking of XP12.
